cmake/Modules/FindQwt.cmake

```diff
@@ -29,7 +29,7 @@ find_path(QWT_INCLUDE_DIRS
 )
 
 find_library (QWT_LIBRARIES
-  NAMES ${PC_QWT_LIBRARIES} qwt6-${QWT_QT_VERSION} qwt-${QWT_QT_VERSION}
+  NAMES qwt ${PC_QWT_LIBRARIES} qwt6-${QWT_QT_VERSION} qwt-${QWT_QT_VERSION}
```

Comment:
In my testing on OSX with both Qwt52 and Qt5Qwt6 installed, putting stock "qwt" here first will result in -any- libqwt that's found being used (in my case, Qwt52), which is not desirable. What we want is for this entry to be last (or at least not first), so that whatever (if anything) PC found is found first, then the other names, then this.

Reply:
Good point! PC must always come first, or it's not being used.

Reply:
What exactly needs to be done here?